network_plugin/network.py

```diff
@@ -120,7 +121,15 @@ def creation_validation(vca_client, **kwargs):
     static_ip = _split_adresses(get_mandatory(net_prop, 'static_range'))
     check_ip(static_ip.start)
     check_ip(static_ip.end)
-    check_ip(get_mandatory(net_prop, "dns"))
+    dns_list = get_mandatory(net_prop, "dns")
```

Comment:
I don't think 'dns' property should be mandatory. For example, if we want to create a private network.